### PR TITLE
Had to remake the branch due to issues.

### PR DESCRIPTION
**Pull request by robertgane**
@rachel-chann Please review my work for any issues in my code :)

### CODE REVIEW

File: prac_05/emails.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            name = input("Name: ")
+        email_to_name[email] = name
+        email = input("Email: ")
+    for email, name in email_to_name.items():
+        print(f"{name} ({email})")
+
+
+def get_name_from_email(email):
+    """Determine possible name from email"""
+    first_part = email.split('@')[0]
```
Comment (ghost) [inline prac_05/emails.py:26]: The variable name "first_part" seems a bit confusing to me. Would it be better to use a name like "prefix"?

File: prac_05/wimbledon.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        for line in in_file:
+            parts = line.strip().split(",")
+            records.append(parts)
+    return records
+
+
+def process_records(records):
+    """Process records into countries set and champion_to_wins dictionary"""
+    countries = set()
+    champion_to_wins = {}
```
Comment (ghost) [inline prac_05/wimbledon.py:34]: Would it be better to use the name "champion_to_count" for this dictionary?

**Review by ghost — COMMENTED** [review]
Well done! I just gave a couple of naming suggestions:)

File: prac_05/emails.py (hunk 1 above)
Comment (robertgane) [reply prac_05/emails.py:26]: I agree

File: prac_05/wimbledon.py (hunk 2 above)
Comment (robertgane) [reply prac_05/wimbledon.py:34]: Great point

**Pull request merged**